vpr/src/base/read_options.cpp

```diff
@@ -1754,6 +1785,97 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("false")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument(args.place_static_move_prob, "--place_static_move_prob")
+        .help(
+            "The Probability of different move in Simulated Annealing."
+            "The order of moves is: {uniformMove, MedianMove, CentroidMove, WeightedCentroid, WeightedMedian, Timing feasible Region(TFR), Critical UniformMove}")
+        .nargs('+')
+        .default_value({"100", "0", "0", "0", "0", "0", "0"})
+
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_static_notiming_move_prob, "--place_static_notiming_move_prob")
+        .help(
+            "The Probability of different non timing move in Simulated Annealing."
+            "The order of moves is: {uniformMove, MedianMove, CentroidMove}")
+        .nargs('+')
+        .default_value({"100", "0", "0"})
+
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_high_fanout_net, "--place_high_fanout_net")
+        .help(
+            "Sets the assumed high fanout net during placement. ")
+        .default_value("10")
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument<bool, ParseOnOff>(args.simpleRL_agent_placement, "--simpleRL_agent_placement")
+        .help(
+            "Uses a simple RL agent in choosing the appropiate move type in placement. ")
+        .default_value("on")
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument<bool, ParseOnOff>(args.place_agent_multistate, "--place_agent_multistate")
+        .help(
+            "Enable multistate agent. "
+            "A second state will be activated late in the annealing and in the Quench that includes all the timing driven directed moves.")
+        .default_value("on")
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument<bool, ParseOnOff>(args.place_checkpointing, "--place_checkpointing")
+        .help(
+            "Enable Placement checkpoints. "
```

Comment:
typo: "agent's 2nd state is activated"
Also, explain what a checkpoint is (save the placement and restore it if it better than later placements).

Reply:
Done.